build.zig

```diff
@@ -160,4 +160,49 @@ pub fn build(b: *Builder) !void {
         "target remote localhost:1234",
     });
     debug_step.dependOn(&debug_cmd.step);
+
+    const zip_folder = "rpi-sdcard";
```

Comment:
I think it would be cleaner to put the build zip/image file into the output directory (`b.cache_root`). This will keep built files in one place.

Reply:
> Good job on getting it working in hardware! You're not going to like me for requesting this but could you add doc comments above each function, struct, struct field and public variable? Documentation is important :) Have a look at src/kernel/filesystem/vfs.zig for inspiration.

Will do.

Reply:
Done: https://github.com/markfirmware/pluto/blob/da14037897a64676ef5fcc4d63e23a102690f842/build.zig#L70